# HG changeset patch
# Parent  95c43eae8b50ec7f811b4283e8165a303f8378a1
Update README and add some build instructions

diff --git a/README.rst b/README.rst
--- a/README.rst
+++ b/README.rst
@@ -33,8 +33,8 @@
 lightsd can target single or multiple bulbs at once:
 
 - by device address;
-- by device label (coming up, labels are already discovered);
-- by tag (coming up, tags are already discovered);
+- by device label;
+- by tag;
 - broadcast;
 - composite (list of targets);
 
@@ -61,4 +61,22 @@
 
 lightsd is actively developed and tested from Arch Linux and Mac OS X.
 
+Build instructions
+------------------
+
+Clone this repository and then:
+
+::
+
+   .../lightsd$ mkdir build && cd build
+   .../lightsd/build$ cmake ..
+   .../lightsd/build$ make -j5
+
+Finally, to start lightsd with the jsonrpc interface listening on localhost
+port 1234:
+
+::
+
+   .../lightsd/build$ core/lightsd -v info -l ::1:1234
+
 .. vim: set tw=80 spelllang=en spell:
